torch/_higher_order_ops/scan.py

```diff
@@ -48,9 +48,8 @@ def scan(
         [pytree.PyTree, pytree.PyTree], Tuple[pytree.PyTree, pytree.PyTree]
     ],
     init: pytree.PyTree,
-    input: pytree.PyTree,
-    dim: int = 0,
-    reverse: bool = False,
+    xs: pytree.PyTree,
```

Comment:
```
def scan(
    combine_fn: Callable[
        [pytree.PyTree, pytree.PyTree], Tuple[pytree.PyTree, pytree.PyTree]
    ],
    init: pytree.PyTree,
    input: pytree.PyTree,
    /,
    dim: int = 0,
    reverse: bool = False,
) -> Tuple[pytree.PyTree, pytree.PyTree]
```
The trick is "/".

Reply:
Thank you, I marked combine_fn, init and xs as position-only.